modules/activiti-engine/src/test/java/org/activiti/engine/test/bpmn/event/signal/ThrowSignalEventSessionManagerTest.java

```diff
@@ -0,0 +1,58 @@
+package org.activiti.engine.test.bpmn.event.signal;
+
+import org.activiti.engine.impl.test.PluggableActivitiTestCase;
+import org.activiti.engine.test.Deployment;
+
+public class ThrowSignalEventSessionManagerTest extends PluggableActivitiTestCase {

```

Comment:
I would like to add the test case that check the case behaviors not supported by this fix.
Specifically, it is a test case for patterns that set the following for each signal event.
1. activiti:async="true"
2. activiti:async="true" AND activiti:exclusive="false"

Reply:
@daisuke-yoshimoto I am not sure this will have any effect on throw, catch and event gateway behaviors, as I am already testing for all async throw events. Any async catch behavior should work out of the box in a separate transaction. I will be happy to execute your test case...

Reply:
@igdianov I'm sorry for the delay.

> as I am already testing for all async throw events. Any async catch behavior should work out of the box in a separate transaction. I will be happy to execute your test case...

I made test cases  not supported by this fix.
Please check them.
https://drive.google.com/file/d/1rKMNohUpNIAqgCCHql2t6P7M_fVCryla/view?usp=sharing
https://drive.google.com/open?id=1wsK60AXiUmhw0rGN-3jAfrnxAscTnjuT
https://drive.google.com/open?id=17KnP88mlbceHceZxE8RgpTzyPSEzwAAi